### DISCUSSION TITLE
Mega palettes with +50 steps on all of them

### DISCUSSION THREAD

**Discussion by jenstornell** [tailwindlabs/tailwindcss#4207] [Ideas]
With JIT there is less to worry about size. A common nag I have is that the colors in the scales are not enough. Often there is a color between two shades "missing".

50
100
150
200
250
...

It means the palettes would be double in size which also means less fiddling with custom values.

**Comment by NikitaIT** [comment]
Use `bg-[#f0f8ff]` or [docs/customizing-colors](https://tailwindcss.com/docs/customizing-colors).

When the set is limited, you don't have 100500 color options by default, which is not good for design. My opinion is it should be user setting. The current Set is small enough to remember and not too big not to know which color to choose.

**Comment by NikitaIT** [comment]
related:
- https://github.com/tailwindlabs/tailwindcss/discussions/2453
